### PR TITLE
feat: exchangeModule triggers error drawer

### PR DESCRIPTION
**Pull request by CremaFR**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Moved errors from swap-live-app to exchange module in order to have the same behavior between Moonpay and swap-live-app

### ❓ Context

[- **12020**: ](https://ledgerhq.atlassian.net/browse/LIVE-12020)

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Review by sprohaszka-ledger — COMMENTED** [review]
Just one quick comment

File: libs/ledger-live-common/src/exchange/swap/types.ts [hunk 1]
```diff
@@ -5,6 +5,14 @@ import { Transaction, TransactionRaw } from "../../generated/types";
 import { Result as UseBridgeTransactionResult } from "../../bridge/useBridgeTransaction";
 import { DeviceModelId } from "@ledgerhq/devices";
 
+export type SwapLiveError = {
```
[the remark this answers is not in the document]
Comment (CremaFR) [reply libs/ledger-live-common/src/exchange/swap/types.ts:8]: Yeah you're right removing this one

**Review by sprohaszka-ledger — APPROVED** [review]
LGTM

File: libs/exchange-module/src/index.ts [hunk 2]
```diff
@@ -107,8 +106,6 @@ export class ExchangeModule extends CustomModule {
     fromAccountId: string;
     toAccountId: string;
     swapId: string;
-    amountExpectedTo: bigint;
-    magnitudeAwareRate: bigint;
```
Comment (cpellizzari-ledger) [inline libs/exchange-module/src/index.ts:111]: not related to this PR, but seen with @CremaFR , was missing from another PR

**Review by cpellizzari-ledger — APPROVED** [review]

**Pull request merged**